src/modules/getData.js

```diff
@@ -0,0 +1,25 @@
+export async function getData() {
+  const response = await fetch('https://api.tvmaze.com/search/shows?q=girls');
+  const myJson = await response.json();
+
+  const likesResponse = await fetch(
+    'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/4aTSjUAMwHbqF73Z3l5i/likes',
+  );
+  const likesJson = await likesResponse.json();
+
+  const result = myJson.slice(2, 8);
+
+  result.forEach((tvObj) => {
+    tvObj.likes = 0;
+  });
+  result.forEach((tvObj) => {
+    likesJson.forEach((likesObj) => {
+      if (+tvObj.show.id === +likesObj.item_id) tvObj.likes = likesObj.likes;
+    });
+  });
+
+  // renderData(result);
```

Comment:
- [ ] As the previous reviewer suggested, we need to follow [Javascript best practices](https://www.w3schools.com/js/js_best_practices.asp). This involves also ensuring that there is no commented code in the codebase. The problem with the commented code is that it's messy and makes it difficult to read. Kindly deleted this commented code and you will be good to go ☺️.